<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Stefan Nikolaus &lt;stefan.nikolaus@kdemail.net&gt;</author>
 <comment>This file is part of the KDE project.
Copyright 2009 Stefan Nikolaus &lt;stefan.nikolaus@kdemail.net&gt;

This program is free software; you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation; either version 2 of the License, or
(at your option) any later version.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License
along with this program; if not, write to the Free Software
Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
</comment>
 <class>Calligra::Sheets::GoalSeekWidget</class>
 <widget class="QWidget" name="Calligra::Sheets::GoalSeekWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>110</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label1">
     <property name="text">
      <string>Set cell:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="Calligra::Sheets::RegionSelector" name="selector1" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label2">
     <property name="text">
      <string>To value:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="Calligra::Sheets::RegionSelector" name="selector2" native="true"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label3">
     <property name="text">
      <string>By changing cell:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="Calligra::Sheets::RegionSelector" name="selector3" native="true"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QWidget" name="outcome" native="true">
     <layout class="QFormLayout" name="formLayout_preview">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label4">
        <property name="text">
         <string>Goal seeking with cell &lt;cell&gt; found &lt;a | no&gt; solution:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Calligra::Sheets::RegionSelector</class>
   <extends>QWidget</extends>
   <header location="global">ui/RegionSelector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
